resources/views/editfaq.blade.php

```diff
@@ -0,0 +1,25 @@
+<?php
+use Illuminate\Support\Facades\DB
+?>
+
+<x-layout>
+    <x-slot:title>
+        Edit FAQ Post {{ $post }}
+    </x-slot>
+    <form method="post" action="/editfaqpost">
+        @method('put')
+        @csrf
+        <h1>Vraag: </h1>
+        <input type="text" name="question" value="{{ DB::table('faq')->where('id', $post)->value('question') }}">
+        <h1>Antwoord: </h1>
+        <input type="text" name="answer" value="{{ DB::table('faq')->where('id', $post)->value('answer') }}">
+        <input type="hidden" name="id" value="{{ $post }}">
+        <br>
```

Comment:
Using `<br>`  tags is not good practice for layout management. Try to experiment with CSS margins or paddings.